README.md

```diff
@@ -79,16 +79,18 @@ For additional help, join us [in our Slack](https://now-examples-slackin-rrirkqo
 
 ### Supported Frameworks
 
--   [React](app/react) [![React](https://img.shields.io/npm/dt/@storybook/react.svg)](app/react)
--   [React Native](app/react-native) [![React Native](https://img.shields.io/npm/dt/@storybook/react-native.svg)](app/react-native)
--   [Vue](app/vue) [![Vue](https://img.shields.io/npm/dt/@storybook/vue.svg)](app/vue)
--   [Angular](app/angular) [![Angular](https://img.shields.io/npm/dt/@storybook/angular.svg)](app/angular)
--   [Polymer](app/polymer) [![Polymer](https://img.shields.io/npm/dt/@storybook/polymer.svg)](app/polymer)
--   [Mithril](app/mithril) <sup>alpha</sup> [![Mithril](https://img.shields.io/npm/dt/@storybook/mithril.svg)](app/mithril)
--   [Marko](app/marko) <sup>alpha</sup> [![Marko](https://img.shields.io/npm/dt/@storybook/marko.svg)](app/marko)
--   [HTML](app/html) <sup>alpha</sup> [![HTML](https://img.shields.io/npm/dt/@storybook/html.svg)](app/html)
--   [Svelte](app/svelte) <sup>alpha</sup> [![Svelte](https://img.shields.io/npm/dt/@storybook/svelte.svg)](app/svelte)
--   [Riot](app/riot) <sup>alpha</sup> [![Riot](https://img.shields.io/npm/dt/@storybook/riot.svg)](app/riot)
+| Framework | Demo latest | Demo prerelease |  |
+|----|---|---|---|
+| [React](app/react) | [v3.4.x](https://release-3-4--storybooks-official.netlify.com), [v3.2.x](https://release-3-3--storybooks-official.netlify.com) | [v4.0.0-alpha](https://storybooks-official.netlify.com) | [![React](https://img.shields.io/npm/dt/@storybook/react.svg)](app/react) |
```

Comment:
I suppose, there should be a 3.3, right?

Reply:
nice catch, will fix